german/metamath.tex

```diff
@@ -2554,7 +2555,7 @@ \section{Ihr erster Beweis}\label{frstprf}
 
 Ein oft nützlicher Befehl ist \texttt{minimize{\char`\_}with*/brief}, der versucht, den Beweis zu verkürzen.  Er kann den Prozess des Beweisens effizienter machen, indem er Sie einen etwas "`schlampigen"' Beweis schreiben lässt und ihn dann durch einige feine Optimierungsdetails für Sie bereinigt (obwohl er keine Wunder vollbringen kann, wie z.B. die Umstrukturierung des gesamten Beweises).
 
-\section{\sloppy Hinweise zur Bearbeitung einer Daten\-basis}
+\section{\texorpdfstring{\raggedright}{}Hinweise zur Bearbeitung einer Datenbasis}
```

Comment:
Is this really working? I get the following error:
`Argument of \@sect has an extra }. ...nweise zur Bearbeitung einer Datenbasis}`

Can \raggedright be used in a \section?

Reply:
It seems to be working for me locally, and the CI run does not show the error message. Do you still get that message when compiling it using `./generate-pdf normal de`?

Reply:
I#m using the tools TeXstudio and MiKTeX to work with (La)TeX files, including the generation of PDF documents. And these tools give me the error. But if it is working with your script/environment, it's fine for me. I would have a closer look on this issue later. So this PR is still accepted by me.

Reply:
The following combination of \sloppy and \texorpdfstring is working:
`\section{\texorpdfstring{\sloppy}{}Hinweise zur Bearbeitung einer Daten\-basis}`

Reply:
I tried that and it produces exactly the same result as nothing at all: the section looks something like:
```
2.5    Hinweise zur Bearbeitung einer Daten-
       basis
````
where the `Daten-` sticks out very slightly. IMO the sticking out isn't bad enough that it seems worth addressing in the first place, which is why my first revision just removed the `\sloppy` entirely.